digdag-docs/src/workflow_definition.rst

```diff
@@ -283,7 +283,7 @@ If ``_retry: N`` (N is an integer: 1, 2, 3, ...) parameter is set to a group, it
       sh>: tasks/analyze_prepared_data_sets.sh
 
 
-Tasks also support ``_retry: N`` parameter to retry the specific task. Note that some operators don't support the generic ``_retry`` option but has its own options to control retrying behavior.
+Tasks also support ``_retry: N`` parameter to retry the specific task. Note that some operators don't support the generic ``_retry`` option but have their own options to control retrying behavior. Operators that involve external systems may reuse previous results. To ensure an external task is repeated, you may use ``_retry`` at the group level.
```

Comment:
@eddsteel Sorry for my late response and thank you for your contribution!

> Operators that involve external systems may reuse previous results
Does it mean operators cache the result? right?
If you know such operator and code, could you explain?

Reply:
I understood the situation. td operator is one of the such operator.